src/main/java/mate/jdbc/model/Car.java

```diff
@@ -0,0 +1,92 @@
+package mate.jdbc.model;
+
+import java.util.List;
+
+public class Car {
+    private Long id;
+    private String model;
+    private Manufacturer manufacturer;
+    private List<Driver> drivers;
+
+    public Car(Long id, String model) {
+        this.id = id;
+        this.model = model;
+    }
+
+    public Car(Long id, String model, Manufacturer manufacturer, List<Driver> drivers) {
+        this.id = id;
+        this.model = model;
+        this.manufacturer = manufacturer;
+        this.drivers = drivers;
+    }
+
+    public Car(String model, Manufacturer manufacturer, List<Driver> drivers) {
+        this.model = model;
+        this.manufacturer = manufacturer;
+        this.drivers = drivers;
+    }
+
+    public Car(String model, Manufacturer manufacturer) {
+        this.model = model;
+        this.manufacturer = manufacturer;
+    }
+
+    public Car(Long id, String model, Manufacturer manufacturer) {
+        this.id = id;
+        this.model = model;
+        this.manufacturer = manufacturer;
+    }
+
+    public Car(String model) {
+        this.model = model;
+    }
+
+    public Car() {
```

Comment:
This empty constructor is unnecessary, as you have other constructors with more fields. You can safely remove this constructor.